memory-db/Cargo.toml

```diff
@@ -11,6 +11,8 @@ heapsize = { version = "0.4", optional = true }
 parity-util-mem = { version = "0.2", default-features = false }
 hash-db = { path = "../hash-db", default-features = false, version = "0.15.2"}
 hashbrown = { version = "0.6.3", default-features = false, features = [ "ahash" ] }
+# There's a compilation error with ahash-0.2.17, which is permitted by the 0.2.11 constraint in hashbrown.
+ahash = "0.2.18"
```

Comment:
As it's not a direct dependency, shouldn't it be fixed in Cargo.lock though?

Reply:
One typically only commits `Cargo.lock`s for applications, not for libraries.

Reply:
Right, I meant in end-user application's `Cargo.lock`, but it's not a big deal anyway.